searchlib/src/vespa/searchlib/queryeval/nearest_neighbor_blueprint.h

```diff
@@ -23,6 +23,7 @@ class NearestNeighborBlueprint : public ComplexLeafBlueprint {
     std::unique_ptr<vespalib::tensor::DenseTensorView> _query_tensor;
     uint32_t _target_num_hits;
     bool _approximate;
+    bool _use_brute_force;
```

Comment:
i don't see the need to have _use_brute_force as another flag; just update _approximate with "false" instead.